tests/update/zypper_up.pm

```diff
@@ -1,23 +1,26 @@
 use strict;
 use base "openQAcoretest";
 use testapi;
+use utils;
 
 sub run {
-    assert_screen "openqa-desktop", 500;
+    wait_for_desktop;
     send_key "ctrl-alt-f2";
     assert_screen "inst-console";
     type_string "root\n";
-    assert_screen "password-prompt", 10;
+    assert_screen "password-prompt";
     type_string "1\n";
```

Comment:
I'm confused here. Don't we have select_console for this? And isn't the password different for 13.2 and Tumbleweed base?